sphinx/domains/python.py

```diff
@@ -1046,6 +1086,7 @@ class PythonDomain(Domain):
         'classmethod':  ObjType(_('class method'),  'meth', 'obj'),
         'staticmethod': ObjType(_('static method'), 'meth', 'obj'),
         'attribute':    ObjType(_('attribute'),     'attr', 'obj'),
+        'property':     ObjType(_('property'),      'prop', 'attr', 'obj'),
```

Comment:
I thought you said you had removed the `prop` role - but isn't that what this line is defining? Or do I misunderstand?

Reply:
What do you mean? This is a definition of object-type, not a role.

Reply:
The documentation of `ObjType` says the arguments are:

> *roles*: all the roles that can refer to an object of this type

Reply:
Ah, okay. understand.